Makefile

```diff
@@ -25,7 +25,11 @@ test-unit:
 
 test-unit-with-coverage:
 	$(call print_msg, Running unit tests with coverage... )
-	nosetests -v --with-xcoverage --cover-package=app test/unit test/integration
+	nosetests -v --with-xcoverage --cover-package=app test/unit
+
+test-integration-with-coverage:
+	$(call print_msg, Running unit tests with coverage... )
+	nosetests -v --with-xcoverage --cover-package=app test/integration
```

Comment:
I had to break these out because travis was spuriously failing my integration tests consistently. I suspect is has something to do with leftover global state.